src/applications/caregivers/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import moment from 'moment';
+import { connect } from 'react-redux';
+import Scroll from 'react-scroll';
+
+import { focusElement } from 'platform/utilities/ui';
+
+const scroller = Scroll.scroller;
+const scrollToTop = () => {
+  scroller.scrollTo('topScrollElement', {
+    duration: 500,
+    delay: 0,
+    smooth: true,
+  });
+};
+
+export class ConfirmationPage extends React.Component {
+  componentDidMount() {
+    focusElement('.schemaform-title > h1');
+    scrollToTop();
+  }
+
+  render() {
+    const { submission, data } = this.props.form;
+    const { response } = submission;
+    const name = data.veteranFullName;
+
+    return (
+      <div>
+        <h3 className="confirmation-page-title">Claim received</h3>
+        <p>
+          We usually process claims within <strong>a week</strong>.
+        </p>
+        <p>
+          We may contact you for more information or documents.
+          <br />
+          <i>Please print this page for your records.</i>
+        </p>
+        <div className="inset">
+          <h4>
+            10-10CG Claim <span className="additional">(Form 10-10CG)</span>
+          </h4>
+          <span>
+            for {name.first} {name.middle} {name.last} {name.suffix}
+          </span>
+
+          {response && (
+            <ul className="claim-list">
+              <li>
+                <strong>Date received</strong>
+                <br />
+                <span>{moment(response.timestamp).format('MMM D, YYYY')}</span>
+              </li>
+            </ul>
+          )}
+        </div>
+      </div>
+    );
+  }
+}
+
+function mapStateToProps(state) {
```

Comment:
We could change this to:
```jsx
const mapStateToProps = (state) => ({
  form: state.form,
});
```
if you want! 🙂

Reply:
Is there any difference other than syntax? I see you're creating a const in this case but I can't figure out if there's any advantage. Also, no need for those parenthesis I believe.

Reply:
Good question Aaron, well it is partially very much a stylistic thing, and the new code we write should be written in the newest adopted format to be consistent. Also the ES6 arrow functions have better scoping which cause less side effects in the code due to scoping. Here is an article that talks about how `this` is handled in each function type.

https://medium.com/@thejasonfile/es5-functions-vs-es6-fat-arrow-functions-864033baa1a

Reply:
We also use a Yeoman generator at VA.gov so I am going to update the code and open a separate PR on the generator so this work will be automated in the future.

Reply:
Love the arrow function and use it almost exclusively on personal stuff. I've run into scoping issues when not all functions in a class are consistent so that's my primary goal but I've been known to refactor existing code to update to newer strategies and patterns.
LOVE Yeoman! Makes me feel like Neo. Wish my current place used a template generator XD.

Reply:
I will say that as far as I know, declaring a const of a lambda provides minimal improvements to memory depending on how the framework handles garbage collection.